### ISSUE TITLE
Add ARIA Accessibility

### ISSUE THREAD

**Issue by t3mp0ralflux** [t3mp0ralflux/magical-kitties-fe#56]
Needs to have accessibility for those that have limited vision. Currently, there are no alternate texts to links, audible links for the hard of seeing, etc.

a/c:
- Site needs to be ADA and ARIA compliant.

**Comment by t3mp0ralflux** [comment]
not needed at this time.
